packages/components/src/Components/TagCount/tagCount.scss

```diff
@@ -0,0 +1,13 @@
+.ins-c-tagCount {
```

Comment:
Filename should be `tag-count.scss` so we are aligned with other components.

Classname should be `.ins-c-tag-count`.